scripts/covid19_india/cases_count_districts_data/test_data/test_data.csv

```diff
@@ -0,0 +1,20 @@
+Date,State,District,Confirmed,Recovered,Deceased,Other,Tested
+26-04-2020,Andaman and Nicobar Islands,Unknown,33,11,0,0,
```

Comment:
Consider demoing this using the previewer too.

https://datacommonsorg.github.io/tools/mcf-browser/#&file=https://raw.githubusercontent.com/datacommonsorg/data/2a6de8fcfd1584c672402229698d93d03e67ef45/scripts/india_lgd/local_government_directory_states/LocalGovernmentDirectory_States.tmcf&file=https://raw.githubusercontent.com/datacommonsorg/data/2a6de8fcfd1584c672402229698d93d03e67ef45/scripts/india_lgd/local_government_directory_states/LocalGovernmentDirectory_States.csv